classes/opponentPhaseCharge.lua

```diff
@@ -0,0 +1,66 @@
+-- The phase where it charges at you
+
+local opponentPhaseCharge = {}
+opponentPhaseCharge.__index = opponentPhaseCharge
+
+
+local isCharging = true
+local chargeUpTime = 1.5
+local chargeTime = 1.5
```

Comment:
Do the same with these two. Don't make local variables, rather, make them members of the object

Reply:
```
self.chargeUpTime = 1.5
self.chargeTime = 1.5
```
both in the .new() function